lib/manageiq_performance/middleware_storage/file.rb

```diff
@@ -46,13 +46,13 @@ def filename env, ext=:data
       end
 
       def format_path_for_filename path
-        request_path = path.to_s.gsub("/", "%")[1..-1]
+        request_path = path.to_s.gsub("/", "%").sub(/^%/, '')
```

Comment:
I am going to assume you are referring to `[1..-1]` versus `.sub(/^%/, '')` (instead of the previous change to us `%` in place of `/` instead of slashes).

[This commit](https://github.com/ManageIQ/manageiq-performance/commit/bfc121d766f1a0e2ff29a6361a94df668c426ff1) was done because the `name` code was being truncated and displayed `names` like `required` as `equired`, which was the incorrect intent.  While I agree that this is probably slightly less performant since a regexp is now used instead of "effectively" a substring, the context is now necessary since everything piped into this method isn't a url path starting with a `/`.